City-of-Helsinki/drupal-helfi
# UHF-30 Hero config

How to test: 
- Check these:
   - https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/13
   - https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/4
   - https://github.com/City-of-Helsinki/drupal-hdbt/pull/24
